header.go

```diff
@@ -811,12 +902,31 @@ func (h *ResponseHeader) VisitAll(f func(key, value []byte)) {
 			f(strSetCookie, v)
 		})
 	}
+	if len(h.trailer) > 0 {
+		visitArgs(h.trailer, func(k, v []byte) {
+			f(strTrailer, k)
+		})
```

Comment:
Shouldn't this just call `f(strTailer, appendArgsKeyBytes(nil, h.trailer, strCommaSpace))`?

Reply:
I still think it's better if this calls `f` once with all the trailer values separated by a comma. This is also how it will go over the wire.

Reply:
Sorry, I forgot to remove `visitArgs`